ServerCore/Pages/Resources/pd2021.2203/FAQPartial.cshtml

```diff
@@ -0,0 +1,130 @@
+    </p>
+
+    <h3>What is a &quot;Metapuzzle&quot;? </h3>
+    <p>A metapuzzle is a special puzzle that takes the progress you've made from solving other puzzles and combines them in some way. It is not necessary to solve all other puzzles before solving the final metapuzzle. The metapuzzle will be clearly labeled.</p>
+
+    <h3>
+        How do we solve puzzles that don't have instructions?
+    </h3>
+    <p>
+        The implicit instruction for all puzzles is &quot;Figure out the final answer&quot;. Puzzles without instructions are either classic puzzle forms, perhaps with a twist, or puzzles where the challenge is figuring out how to solve the puzzle. In the latter case, the key is usually to look for patterns or common bonds between the parts of the puzzle. For example, are many of the words spelled wrong? Have all the vowels been left out? Sometimes there may be a hint hidden in the puzzle.
+    </p>
+
+    <h3>
+        What's the best way to win?
+    </h3>
+    <p>
+        The best way to win is to solve puzzles quickly so that you get to the final metapuzzle. The best way to accomplish this is with teamwork. Many teams prefer to work individually on the easier puzzles and work as a group on the harder puzzles. If you get stuck, ask for help or switch to another puzzle. Make sure you keep track of which puzzles you've already solved and confirmed using the answer submission system.
+    </p>
+
+    <h3>
+        What's the best way to lose?
+    </h3>
+    <p>
+        Not having fun. If you're not having fun working on a particular puzzle, switch to another one.
+    </p>
+
+    <h3>
+        What if we're really stuck?
+    </h3>
+    <p>
+        If you don't understand how to solve a puzzle, contact Puzzleday staff. We may give you a hint or clarify the instructions. Aside from that, there are enough puzzles that you probably won't be stuck on all of them.
+    </p>
+
+    <h3>
+        What are the prizes?
+    </h3>
+    <p>
+        Each member of the winning teams will receive some digital theme-related prizes.
+    </p>
+
+    <h3>
+        Can I finish by accident, without solving the puzzles?
+    </h3>
+    <p>
+        Haha. No.
+    </p>
+
+    <h3>
+        Do I have to register to participate?
+    </h3>
+    <p>
+        Yes. If you do miss the registration deadline, send mail to Puzzleday staff at <a href="mailto:puzzhunt@microsoft.com">puzzhunt</a>.
+    </p>
+
+    <h3>
+        Do I have to be on the Redmond campus to participate?
+    </h3>
+    <p>
+        No! This year, we will be hosting the opening and closing online over Teams, and each team will collaborate with each other on the digital platform of their choosing. No travel will ever be required.
```

Comment:
I should hope not, if most players are on another continent lol

Reply:
I definitely thought about leaving the line in about "even if you're around, don't visit us", but figured they'd probably be a little too far away to visit...